CHANGELOG.next.asciidoc

```diff
@@ -48,6 +48,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 *Metricbeat*
 
 - Setting period for counter cache for Prometheus remote_write at least to 60sec {pull}38553[38553]
+- Added a conditional check to see if the statsd metric contains ',' or ';' and split accordingly. {pull}39619[39619]
```

Comment:
```suggestion
- Add support of Graphite series 1.1.0+ tagging extension for statsd module. {pull}39619[39619]
```

Reply:
Also, I think we should update README. See an example: https://github.com/prometheus/statsd_exporter?tab=readme-ov-file#tagging-extensions

@ritalwar @tehbooom What do you think? Else, one has to go to code to figure it out to see what extensions are supported.

Reply:
@ritalwar @shmsr How do we feel about adding this to the README?

```asciidoc
[float]
=== Suported tag extensions

The `statsd` module supports the following tags:

https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#the-dogstatsd-protocol[DogStatsD]

`<metric name>:<value>|<type>|@samplerate|#<k>:<v>,<k>:<v>`

https://github.com/influxdata/telegraf/blob/master/plugins/inputs/statsd/README.md#influx-statsd[InfluxDB]

`<metric name>,<k>=<v>,<k>=<v>:<value>|<type>|@samplerate`

https://graphite.readthedocs.io/en/latest/tags.html#graphite-tag-support[Graphite 1.1.x]

`<metric name>;<k>=<v>;<k>=<v>:<value>|<type>|@samplerate`
```

Reply:
Perhaps the heading could be like:

- "DogStatsD-style tags"
- "InfluxDB-style tags"

and so on.

Reply:
I'm fine with just listing the names, but we can add something like:

Supported Tag Extensions

Examples of tag styles supported by the `statsd` module:

- DogStatsD 
`<metric name>:<value>|<type>|@samplerate|#<k>:<v>,<k>:<v>`
- InfluxDB
`<metric name>,<k>=<v>,<k>=<v>:<value>|<type>|@samplerate`
- Graphite 1.1.x
`<metric name>;<k>=<v>;<k>=<v>:<value>|<type>|@samplerate`